python/neuroglancer/pipeline/storage.py

```diff
@@ -100,27 +65,45 @@ def extract_path(cls, layer_path):
         else:
             return cls.ExtractedPath(*match.groups())
 
-    def put_file(self, file_path, content, compress=True):
+    def put_file(self, file_path, content, compress=False):
```

Comment:
Oh, I can restructure it more, but that block parameter isn't meant to be used in user code. `get_files` and `put_files` are intended to read synchronously while put_file is default async. Though I think get_file is synchronous. :P